### PR TITLE
Enable output from Installer and all machines in Processes panel

### PR DESCRIPTION
**Pull request by azatsarynnyy**
### What does this PR do?
Enables displaying all logs from Installer and machines in Process panel.

### What issues does this PR fix or reference?
#5175 

#### Changelog
N/A

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    private String getWsMasterURL() {
+        String protocol = "https:".equals(getProtocol()) ? "wss://" : "ws://";
+        String host = getHost();
+        String context = "/wsmaster/output/websocket/";
+
+        return protocol + host + context + appContext.getAppId();
+    }
+
+    public void terminate() {
+        initializer.terminate("ws-master");
```
Comment (dkulieshov) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java]: I guess this should be `"ws-master-output"`, no?

**Review by dkulieshov — APPROVED** [review]

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java (hunk 1 above)
Comment (vparfonov) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java]: @azatsarynnyy looks like @dkuleshov assumption is correct?

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/events/WorkspaceEventsUnsubscriber.java [hunk 2]
```diff
@@ -37,6 +34,8 @@
             subscriptionManagerClient.unSubscribe("ws-master", "workspace/statusChanged", scope);
             subscriptionManagerClient.unSubscribe("ws-master", "machine/statusChanged", scope);
             subscriptionManagerClient.unSubscribe("ws-master", "server/statusChanged", scope);
+            subscriptionManagerClient.unSubscribe("ws-master-output", "machine/log", scope);
```
Comment (vparfonov) [inline ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/events/WorkspaceEventsUnsubscriber.java]: We use those value in several place. Do we need some constants  for them?

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java (hunk 1 above)
Comment (azatsarynnyy) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/jsonrpc/WsMasterOutputJsonRpcInitializer.java]: yep. thanks!

File: ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/events/WorkspaceEventsUnsubscriber.java (hunk 2 above)
Comment (azatsarynnyy) [reply ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/events/WorkspaceEventsUnsubscriber.java]: sure, I'll move endpoint ids to the shared constants in the subsequent PRs

**Review by vparfonov — APPROVED** [review]

**Pull request merged**